src/public/Interfaces.ts

```diff
@@ -9,14 +9,20 @@
  * @returns Output bindings (optional). If you are returning a result from a Promise (or an async function), this 
  * result will be passed to JSON.stringify unless it is a string, Buffer, ArrayBufferView, or number.
  */
-export type AzureFunction = ((context: Context, ...args: any[]) => Promise<any> | void);
+export type AzureFunction<TReturn = unknown> = ((context: Context, ...args: any[]) => Promise<TReturn> | void);
 
+/**
+ * Context bindings object. Provided to your function binding data, as defined in function.json.
+ */
 export interface ContextBindings {
-  [name: string]: any;
+  [name: string]: unknown;
```

Comment:
changing from any -> unknown is a breaking change.
This will encourage customers to extend public interfaces to include properties they are using in the code.

Reply:
Do you want to change any other `any`s? as theres a couple of others in this file :)

Reply:
After chatting with @mhoeger offline we decided to leave it as `any`. Changing to `unknown `will be quite hard and we do not want to introduce breaking changes for the customers code.